grama/tran_pivot.py

```diff
@@ -0,0 +1,255 @@
+__all__ = [
+    "tran_pivot_longer",
+    "tf_pivot_longer",
+    "tran_pivot_wider",
+    "tf_pivot_wider",
+]
+
+import numpy as np
```

Comment:
full-package import; make sure to import only those utilities necessary for this file